spark/pom.xml

```diff
@@ -56,12 +56,12 @@
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
-            <version>2.11.8</version>
+            <version>2.12.8</version>
```

Comment:
you have changed scala version from 2.11.8 to 2.12.8 in pom.xml
Should you use ${scala.version} here

Reply:
resolved